docs/PRD-multi-payment-access.md

```
@@ -0,0 +1,282 @@
+	invoices: InvoiceWithPaymentsAndOwner[];
+	festival: FestivalWithDates;
+};
+```
+
+---
+
+## 7. Server Actions y Queries
+
+Definidas en `app/data/invoices/actions.ts`. Todas son `"use server"`.
+
+#### `fetchReservationsWithInvoicesByProfileAndFestival(profileId, festivalId)`
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)